spec.md

```diff
@@ -480,7 +480,7 @@ Client and registry implementations SHOULD expect to be able to support manifest
 
 ###### Pushing Manifests with Subject
 
-When processing a request for an image or artifact manfiest with the `subject` field, a registry implementation that supports the [referrers API](#listing-referrers) MUST respond with the response header `OCI-Subject: <subject digest>` to indicate to the client that the registry processed the request's `subject`.
+When processing a request for an image manifest with the `subject` field, a registry implementation that supports the [referrers API](#listing-referrers) MUST respond with the response header `OCI-Subject: <subject digest>` to indicate to the client that the registry processed the request's `subject`.
```

Comment:
```suggestion
When processing a request for a manifest with the `subject` field, a registry implementation that supports the [referrers API](#listing-referrers) MUST respond with the response header `OCI-Subject: <subject digest>` to indicate to the client that the registry processed the request's `subject`.
```

Reply:
We haven't approved the subject field on the index manifest yet.

Reply:
ok.. either way is there benefit to restricting the MUST to image manifest... noting the section title is the broader Manifests

Reply:
Wouldn't that require registries to parse manifests for a field that may not have been included in the image-spec?

Reply:
yes I believe so.. Though my generalization was more to reduce complexity in the wording here for when/if subject is added to index...  I didn't contemplate that this push MUST was an implied but only for genuine oci.image.manifest.v1+json manifests with said subject field (implying as of v1.1.0).. hmm

Reply:
I'm with Brandon on this specific one, it would introduces another ambiguity. Also bordering on scope creep here, my aim was to not say "artifact manifest in the document", the previous version did explicitly list the manifests it applied to.